app/.env.symlink

```diff
@@ -1 +0,0 @@
-../.env
```

Comment:
don't remove this file

Reply:
@ashutoshc8101 Done

Reply:
The change shouldn't be reflected here. We have handled this multiple times now.